### DISCUSSION TITLE
overrides in compose API

### DISCUSSION THREAD

**Discussion by dagap** [facebookresearch/hydra#2175] [Q&A]
I asked this on OmegaConf repo as well but I think it is a bit inconvenient to only have this `dot-list` format for the overrides in the hydra compose API (https://hydra.cc/docs/advanced/compose_api/).

Maybe I have missed it but is it possible to generate the `dot-list` representation from an input YAML or dict using either hydra or OmegaConf?

**Comment by Jasha10** [comment]
See my follow-up question [here](https://github.com/omry/omegaconf/discussions/904#discussioncomment-2659357).
